js/actions/webviewActions.js

```diff
@@ -81,6 +81,31 @@ const webviewActions = {
         webview.executeJavaScript('document.webkitRequestFullscreen()')
       }
     }
+  },
+
+  findInPage (searchString, caseSensitivity, forward, webview) {
+    webview = webview || getWebview()
+    if (!webview) {
+      return
+    }
+
+    if (searchString) {
+      webview.findInPage(searchString, {
+        matchCase: caseSensitivity,
+        forward: forward !== undefined ? forward : true,
+        findNext: forward !== undefined
+      })
+    } else {
+      webview.stopFindInPage('clearSelection')
+    }
+  },
+
+  stopFindInPage (webview) {
```

Comment:
fyi @bbondy - the problem with this turned out to be the result of calling stopFindInPage on hide. `keepSelection` doesn't actually keep any state in the webcontents, it just keeps the selection highlighted. If you want to keep the find state you can't call `stopFindInPage`